Source/ThirdParty/libwebrtc/Source/webrtc/modules/desktop_capture/linux/wayland/xdg_desktop_portal_utils.cc

```diff
@@ -186,7 +186,7 @@ void TearDownSession(absl::string_view session_handle,
     g_object_unref(cancellable);
   }
 
-  if (proxy) {
+  if (!!proxy) {
```

Comment:
You're just converting variables named `proxy` regardless of whether it's actually needed...? This is a third-party lib too.

Reply:
I closed this for a reason and these commits got pushed afterwards. 

I'm looking into real solutions before I reopen.